Source/Core/Core/PrimeHack/Mods/FpsControls.h

```diff
@@ -53,8 +53,7 @@ class FpsControls : public PrimeMod {
   void add_control_state_hook_mp3(u32 start_point, Region region);
   // Very large code, apologies for anyone who reads this
   // corresponding assembly is in comments :)
-  void add_strafe_code_mp1_ntsc();
-  void add_strafe_code_mp1_pal();
+  void add_strafe_code_mp1_gc(Region region, u32 calculate_side_move_address, u32 apply_strafe_force_address, u32 disable_rotation_address, u32 clamp_cur_xy_vel_address, u32 max_speed_array_address);
```

Comment:
I appreciate that we include the function addresses normally as arguments, but the seeing as there are so many arguments here you may want to consider adding these to the address db instead. I don't mind including them as arguments normally (when there's only 2 or 3), but originally this was done because we did not have the addressdb until relatively recently.